opslevel/provider.go

```diff
@@ -161,6 +161,7 @@ func (p *OpslevelProvider) Configure(ctx context.Context, req provider.Configure
 		tflog.Error(ctx, fmt.Sprintf("OpsLevel client validation error: %s", err))
 	}
 	tflog.Debug(ctx, "OpsLevel client is valid")
+	opslevel.Cache.CacheTeams(client)
```

Comment:
@davidbloss do you know if this function is only called once for each `terraform plan` or `terraform apply` and not once per parallel resource action?

Reply:
I haven't tested this out but I'm betting this is only called once per plan/apply

Reply:
opted to go with a inline call to GetTeamWithAliases